### DISCUSSION TITLE
USING m5 STACK camera module , having issue in initializing SD card

### DISCUSSION THREAD

**Discussion by sunilkumar105** [jomjol/AI-on-the-edge-device#2649] [General] [answered]
Hey there, I am using [M5STACK ,camera module ](https://docs.m5stack.com/en/unit/ESP32CAM-PSRAM) This module is using ESP32 wrover module as controller and the issue i am facing is It is throwing an error of **SD card init failed.**

**MAIN ISSUE**
CONFLICTING GPIO ::> According to the documentation of M5STACK Cam module, it is using GPIO_NUM_15 with camera, and SDMMC library is also using GPIO15 as CMD pin. 
in the [documentation of sdmmc](https://github.com/espressif/esp-idf/blob/master/examples/storage/sd_card/sdmmc/README.md) i found GPIO pins cant be customised in case of ESP32 which also applies of rthe wrover module i am using. 
So the only possible way to solve this issue is using SDCARD in SPI mode

**QUESTION ::> Has the implementation of using SD card in SPI module is done in this project ??**
**Is there any possible way ( Other than using SD card in SPI MODE ) to solve this ISSUE ?
CAN WE USE SDMMC library FOR using SD CARD IN SPI MODE?**

**Comment by caco3** [comment, marked as answer]
This hardware is not supported as of now!

**Reply by sunilkumar105** [reply]
OK, THNX FOR YOUR RESPONSE
